publish2cloud.py

```diff
@@ -278,18 +284,38 @@ def publish_to_remote_settings(config, section, chunknum):
         'Name': list_name,
         'Checksum': chunk_file['checksum'],
         'Version': chunknum,
+        # The default master branch is the latest list in shavar-prod-lists, we use filter_expression
+        # to make sure only the latest fx versions use this list by setting the expression to greater than
+        # the "latest_supported_version" + 1, since the latest_supported_version is the highest version number in
+        # the shavar prod lists branch names
+        'filter_expression': f'env.version|versionCompare("{shared_state.latest_supported_version+1}") >= 0'
```

Comment:
is this because the master is nightly? I'm a bit confused what i would have to write in the update

Reply:
Once we create the version branch for the Nightly channel, Nightly will use the created version list but not the master branch. Any new updates to the master branch won't apply to Nightly directly, so we need to backport the changes to the version branch of Nightly. This would work, but it's unnecessary. We can avoid creating the version branch for Nightly.